CityPuzzle/Rest Services/Client/HttpClientRequest.cs

```diff
@@ -13,9 +13,19 @@ public class HttpClientRequest
 
         public async Task<string> SendCommand(string objectPath)
         {
-            Console.WriteLine("url + objectPath"+ url + objectPath);
-            var content = await httpClient.GetStringAsync(url + objectPath);
-            return content;
+            Console.WriteLine("url + objectPath" + url + objectPath);
+            Task<string> sendcommand = httpClient.GetStringAsync(url + objectPath);
+            Thread timer = new Thread(new ThreadStart(startTimer));
+            timer.Start();
+            while (timer.IsAlive)
+            {
+                if (sendcommand.IsCompleted)
+                {
+                    return sendcommand.Result;
```

Comment:
after return the startTimer thread is not destroyed so after 3s it throws exception no matter what